supervisor-services-labs/rabbitmq-operator/rabbitmq-operator.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
```

Comment:
remove the field?

Reply:
Since this file is a direct output of the `kctrl` command, I did not modify it. I will check if deleting this filed does not cause any errors with kapp controller on the supervisor.